src/components/BreedCard.vue

```diff
@@ -1,58 +1,72 @@
 <template>
-  <li
-    class="result__list"
-    @click="toggleDetails()"
-    :class="!isActive ? 'hidden-result__details' : 'result__list'"
-  >
-    <div class="result__heading-container">
-      <span class="result__id">{{ id + 1 }}.</span>
-      <h3 class="heading-tertiary">{{ breed.name }}</h3>
-    </div>
+  <router-link :to="`/breeds/${breed.name}`">
```

Comment:
predlagam da uporabljaš linkanje na route name in ne na absoluten path, ker če boš kdaj spreminjal route path pol moreš na večih mestih popravt.. 

`:to="{name: 'nekineki'}"`

Reply:
> par malenkosti za popravt..
> 
> @Luka85 a to da ti poscrolla na iskan breed že dela?

Dela, če kliknem na card, če pa vnesem v url pa ne poscrolla do breeda